Java/syntax_test_java.java

```diff
@@ -603,7 +788,28 @@ void annotatedArgs(@NonNull final String p1,
 // <- meta.annotation.java meta.annotation.parameters.java punctuation.section.parens.end.java
 @fully.qualified.Annotation
 // <- punctuation.definition.annotation.java
-//^^^^^^^^^^^^^^^^^^^^^^^^^ variable.annotation.java
+//^^^^^^^^^^^^^^^^^^^^^^^^^ meta.annotation.java meta.annotation.identifier.java meta.path.java
+//^^^^ variable.annotation.package.java
+//    ^ punctuation.accessor.dot.java
+//     ^^^^^^^^^ variable.annotation.package.java
+//              ^ punctuation.accessor.dot.java
+//                ^^^^^^^^^ variable.annotation.java
+@fully.qualified.ParentClass.InnerAnnotation
```

Comment:
Another comment regarding the one above.

If I understand what #737 says about annotations, only the last part of the qualifier should be marked with `variable.annotation.java`. In this case `InnerAnnotation`.

Reply:
The idea behind the funny scope of the annotation's package is to keep the complete identifier the same color. Same with `entity.other.inherited-class`. Both of these kind of stand out from the rest of the code and with one color this behavior is more 'emphasized'. But that's my personal view, maybe I'm missing something.